core/float/shared/to_i.rb

```diff
@@ -1,5 +1,6 @@
 describe :float_to_i, shared: true do
   it "returns self truncated to an Integer" do
+    -> { (0.0 / 0.0).send(@method) }.should raise_error(FloatDomainError)
```

Comment:
Could you move this to a separate `it` example?
In general we shouldn't mix normal and exceptional cases, and also this behavior does not match the example description.

Reply:
Good point, will do!